amoro-common/src/main/java/org/apache/amoro/process/TableProcessState.java

```diff
@@ -30,6 +30,7 @@ public class TableProcessState implements ProcessState {
   @StateField private volatile long id;
   private final Action action;
```

Comment:
I see the action field is materialized in the DB?

Reply:
Yes
What's the problem?

Reply:
Need to add `@StateField` for action field?

Reply:
The field shoud never change. so action and identifer is unnecessary for @StateField

Reply:
I see